src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -45,38 +48,56 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
   }
 
   public createWallet({ name }: IWalletModalData): void {
-    this.myWalletsService.createWallet(MyWallet.create({ name })).subscribe( (wallet: MyWallet) => {
-        this.myWalletsData.data = [wallet, ...this.myWalletsData.data!]
+    this.myWalletsService.createWallet(MyWallet.create({ name })).subscribe({
+        next: (wallet: MyWallet) => {
+            this.myWalletsData.data = [wallet, ...this.myWalletsData.data!];
+        },
+        error: () => {
+            this.systemNotificationsService.showNotification({ message: 'Some server error during creating', type: NotificationType.Error });
+        },
     });
   }
 
   public updateWallet({ id }: MyWallet, { name }: IWalletModalData): void {
-    this.myWalletsService.updateWallet(MyWallet.create({ id: id!, name })).subscribe( (updatedWallet: MyWallet) => {
-        this.myWalletsData.data = this.myWalletsData.data!.map(walletItem => {
-            if (walletItem.id === id) {
-                return updatedWallet;
-            }
-
-            return walletItem;
-        });
+    this.myWalletsService.updateWallet(MyWallet.create({ id: id!, name })).subscribe({
+        next: (updatedWallet: MyWallet) => {
+            this.myWalletsData.data = this.myWalletsData.data!.map(walletItem => {
+                if (walletItem.id === id) {
+                    return updatedWallet;
+                }
+                return walletItem;
+            });
+        },
+        error: () => {
+            this.systemNotificationsService.showNotification({ message: 'Some server error during updating', type: NotificationType.Error });
```

Comment:
I bet you won't like it but you should now extend tests for all those methods - to test that notifications was shown when there was an error when calling service's method

Reply:
; (